custom_components/csnet_home/api.py

```diff
@@ -175,15 +180,17 @@ async def async_set_temperature(self, zone_id, parent_id, **kwargs):
         data = {
             "orderStatus": "PENDING",
             "indoorId": parent_id,
-            "_csrf": "9d63b828-f229-402f-899c-19dc40b5e447",
+            "_csrf": "be186598-c4d0-4d16-80f0-dc2ab35aad23",
```

Comment:
Not sure changing the CSRF here will be the correct solution... I think it can brake what we have with @MakeTotalSense.
I'm going to make a try.... but what you are proposing in the issue #26 it should be the right way to go: recover these statics values somewhere else.
As I said, it is something I wasn't able to find in my test calls, so I proposed this first "work on my laptop" version :)

Reply:
I think the cookie value should be the same for each call after the login, that's why I've changed that.
I see that in some cases you use a different value.
The best way should anyway be to get that from the cookie after the login

Reply:
Yes... on my side I don't have the same values for the differents API calls after the login... no idea why. I will test this PR on my side to check it out

Reply:
Thank you again for all your effort
If you need we can connect on discord for further discussions